inc/vcf/error.hpp

```diff
@@ -36,14 +55,9 @@ namespace ebi
                   message{message} {}
 
 
-        size_t get_line() const
-        {
-          return line;
-        }
-        const std::string &get_raw_message() const
-        {
-          return message;
-        }
+        size_t get_line() const { return line; }
+        const std::string &get_raw_message() const { return message; }
+        virtual ErrorCode get_code() const { return ErrorCode::error; }
```

Comment:
Could this be made an attribute, same as `line` and `message`, and initialized in the constructor? It would be more consistent with the rest of the class and also prevent virtual calls.